packages/providers/src/core/PoapCompass/PoapCompass.ts

```diff
@@ -167,6 +193,6 @@ export class PoapCompass implements CompassProvider {
  * @property {string} [baseUrl] - Optional base URL for the POAP API. If not provided, a default will be used.
  */
 export interface PoapCompassConfig {
-  apiKey: string;
+  apiKey?: string;
```

Comment:
What's the intention behind this change?

API key will be required 100% of the time.